code/rdb/rdbstandard.q

```diff
@@ -1,6 +1,32 @@
-// Get the relevant RDB attributes
-.proc.getattributes:{`date`tables!(.rdb.rdbpartition[],();tables[])}
-
+\d .proc
+/- Get the relevant RDB attributes
+getattributes:{
+    timecolumns:1!except[(::),gettimecolumns each tables[`.];(::)];
+    attrtable:`date`tables`procname!
+	({[timecolumns]
+	  d:exec(min;max)@\:distinct`date$first each raze value each timecolumns from timecolumns;
+	  first[d]+til 1+last deltas d}[timecolumns];
+	tables[];
+	.proc.procname);
+    if[.ds.datastripe;
+	filtermap:flip @[.j.k read1 hsym`$getenv[`KDBAPPCONFIG],"/striping.json";`$string first .ds.segmentid];
+	filtermap:([]tablename:key filtermap)!([]wc:value filtermap);
```

Comment:
I think you're doing this so that you can use `lj` below, but maybe that's unnecessary?

You should be able to use the dictionary directly something like

```q
update wc:filtermap tablename from timecolumns
```